docs/faq.md

```diff
@@ -28,7 +28,7 @@ ksqlDB requires only:
 We recommend using [Confluent
 Platform](https://www.confluent.io/product/confluent-platform/) or
 [Confluent Cloud](https://www.confluent.io/confluent-cloud/) to run
-{{ site.ak }}.
+{{ site.ak }}. ksqlDB supports Java 8 and 11.
```

Comment:
Out of interest, where did you get this information? We only compile on java 8, but I thought any jdk runtime above 8 should be able to run the compiled JAR (I have not verified).